src/contentType.ts

```diff
@@ -15,13 +15,16 @@ export const DEFAULT_MIME_TYPE = 'text/html'
  * The order is important on this one.
  */
 const tests: Array<(input: testInput) => testOutput> = [
+  // svg
+  async ({ bytes }): testOutput => /^(<\?xml[^>]+>)?[^<^\w]+<svg/ig.test(
+    new TextDecoder().decode(bytes.slice(0, 64)))
+    ? 'image/svg+xml'
+    : undefined,
```

Comment:
This regex is not super easy to maintain, nor parse, nor extend.

Since this is our last fallback prior to using the default mime type, we should make it more extensible (could be xml/svg or even something else in the future).

Ideally, also RegExp functions for parsing XML is usually not a good idea.